ts/spi/provider_frame.ts

```diff
@@ -216,6 +220,16 @@ export class ProviderFrame {
         requestId, this.providerConfig.delegateToBrowser));
   }
 
+  private async handleDisableAutoSignInRequest(requestId: string) {
+    try {
+      await this.localStateProvider.setAutoSignInEnabled(
+          this.providerConfig.clientAuthDomain, false);
+    } catch (e) {
+      // TODO(tch): Handle differently when the localStateProvider has an error?
```

Comment:
So, I think it's probably safe to just log this error and return. Otherwise, we'll force error handling onto the client, and it's not clear what a client could realistically do in this situation.